instruments/instruments/util_fns.py

```diff
@@ -54,6 +54,41 @@ def assume_units(value, units):
         value = pq.Quantity(value, units)
     return value
 
+
+def convert_temperature(temperature, base):
+    """
+    convert the temperature to the specified base
+    :param temperature: a quantity with units of Kelvin, Celsius, or Fahrenheit
+    :type temperature: `quantities.Quantity`
+    :param base: a temperature unit to convert to
+    :type base: `unitquantity.UnitTemperature`
+    :return: the converted temperature
+    :rtype: `quantities.Quantity`
+    """
+    # quantities reports equivalence between degC and degK, so a string comparison is needed
```

Comment:
I already talked to the pyquantities people earlier today; they said that introducing offsets into scaling is outside the scope of the project.

Since C = K + offset, pyquantities sees C=K. Setting these as not equal would require significant work for them, so it won't be resolved.